src/nccl_ofi_api.c

```diff
@@ -173,7 +173,7 @@ ncclResult_t nccl_net_ofi_listen(int dev_id, void *handle, void **lComm)
 
 ncclResult_t nccl_net_ofi_listen_v4(int dev, void* handle, void** listenComm)
 {
-        nccl_net_ofi_conn_handle_t nccl_net_ofi_handle = {0};
+        nccl_net_ofi_conn_handle_t nccl_net_ofi_handle = {};
```

Comment:
TIL that this is only actually legal as of C23.  But AL2's gcc (the oldest of the ones we support) seems ok with it without warning, so I'm good with it.